Description: Hide the mouse cursor on startup and when starting an user session.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Origin: Ubuntu

diff -Nur -x '*.orig' -x '*~' gdm-3.0.0//daemon/gdm-simple-slave.c gdm-3.0.0.new//daemon/gdm-simple-slave.c
--- gdm-3.0.0//daemon/gdm-simple-slave.c	2011-06-05 20:49:08.000000000 +0200
+++ gdm-3.0.0.new//daemon/gdm-simple-slave.c	2011-06-05 21:31:18.087605694 +0200
@@ -1216,9 +1216,6 @@
         /* Put cursor out of the way on first head */
         gdm_slave_set_initial_cursor_position (GDM_SLAVE (slave));
 
-        /* Set the busy cursor */
-        gdm_slave_set_busy_cursor (GDM_SLAVE (slave));
-
         /* The root window has a background that may be useful
          * to cross fade or transition from when setting the
          * login screen background.  We read it here, and stuff
diff -Nur -x '*.orig' -x '*~' gdm-3.0.0//gui/simple-greeter/gdm-greeter-login-window.c gdm-3.0.0.new//gui/simple-greeter/gdm-greeter-login-window.c
--- gdm-3.0.0//gui/simple-greeter/gdm-greeter-login-window.c	2011-06-05 20:49:08.000000000 +0200
+++ gdm-3.0.0.new//gui/simple-greeter/gdm-greeter-login-window.c	2011-06-05 21:32:51.587742830 +0200
@@ -174,7 +174,7 @@
 {
         GdkCursor *cursor;
 
-        cursor = gdk_cursor_new (GDK_WATCH);
+        cursor = gdk_cursor_new (GDK_BLANK_CURSOR);
         gdk_window_set_cursor (gtk_widget_get_window (GTK_WIDGET (login_window)), cursor);
         gdk_cursor_unref (cursor);
 }
